package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1362,7 +1362,7 @@ class Bonds(_Connection):
 
     Must be initialised by a list of zero based tuples.
     These indices refer to the atom indices.
-        Eg:  [(0, 1), (1, 2), (2, 3)]
+    Eg:  [(0, 1), (1, 2), (2, 3)]
```

Comment:
"E.g.," contains periods and is followed by a comma.  think it makes more sense to show the list of tuples as code.
```reST
E.g., ` [(0, 1), (1, 2), (2, 3)]`
```